test.py

```diff
@@ -61,21 +61,24 @@ def test_get_experiment_info(self):
             )
 
     def test_get_experiment_queue(self):
-        test_queues = tuple({
-            '1': {'pipeline': 'main',
-                'expid': {
-                    'log_level': 30,
-                    'class_name': None,
-                    'arguments': {'user': 'QuIQCL', 'time': 1.0, 'save': False, 'color': 'r'},
-                    'file': 'DIRECTORY-PATH'
-                },
-                'priority': 1,
-                'due_date': None,
-                'flush': False,
-                'status': status,
-                'repo_msg': None
-            }
-        } for status in ['pending', 'preparing', 'running', 'run_done', 'analyzing', 'deleting'])
+        test_queues = tuple()
+        for status in ["pending", "preparing", "running", "run_done", "analyzing", "deleting"]:
+            test_queues += ({
```

Comment:
Well, this is a performance or style issue, but `tuple` is designed for an immutable iterable.
If you want to mutate a data structure, `list` is preferred, i.e., `list.append()`.

In more detail, `tuple +` will make a new `tuple` object, so it will allocate a new memory space for each tuple addition (and the new space increases: 0, 1, 2, ...).
On the other hand, `list` will allocate some extra memory space in advance, so new memory allocations occur less frequently.

If you want to use `tuple`, you can wrap it at the end, but I don't think it is necessary.

Furthermore, I think you can just directly iterate over `status` instead of `queue` in the `with` block below. Then you don't have to define `test_queues`.

Please consider these idea and tell me your opinions!